.github/workflows/build_and_test.yml

```diff
@@ -57,6 +57,9 @@ jobs:
           - 'package.json'
           - 'rust/bridge/node/**'
           - 'yarn.lock'
+          ios:
+          - *all
+          - 'rust/bridge/ffi/**'
           swift:
           - *all
           - 'rust/bridge/ffi/**'
```

Comment:
If you add this, let's make it a reference inside `swift`, so there's no duplication.